cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/handler/async/runnable/LoadRunnable.java

```diff
@@ -64,13 +66,29 @@ protected BrokerStats getResult() throws Exception {
         return cachedBrokerStats;
       }
     }
+    if (_populateDiskInfo && !isClusterUsingJBOD()) {
+      throw new UserRequestException("populate_disk_info = true when Kafka cluster is NOT using JBOD");
```

Comment:
Can we refrain from (1) using hardcoded parameter names and (2) emphasis via capitalization -- i.e. maybe we can use something like:
```
throw new UserRequestException(String.format("Cannot set %s=true for non-JBOD Kafka clusters.", POPULATE_DISK_INFO_PARAM));
```

Reply:
Good idea